reportportal_client/service.py

```diff
@@ -213,6 +213,23 @@ def start_test_item(self, name, start_time, item_type, description=None,
         logger.debug("start_test_item - Stack: %s", self.stack)
         return item_id
 
+    def update_test_item(self, description=None, tags=None):
+        """Updates test item
```

Comment:
This docstring does not correspond to PEP257.
D400, First line should end with a period. 
D401, First line should be in imperative mood.

Reply:
Fixed